plugins/repository-gcs/qa/google-cloud-storage/src/test/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageTestServer.java

```diff
@@ -301,6 +425,37 @@ public Response handle(final String method,
             return newResponse(RestStatus.OK, emptyMap(), buildObjectResource(destBucket.name, dest, sourceBytes));
         });
 
+        // Rewrite Object
+        //
+        // https://cloud.google.com/storage/docs/json_api/v1/objects/rewrite
+        handlers.insert("POST " + endpoint + "/storage/v1/b/{srcBucket}/o/{src}/rewriteTo/b/{destBucket}/o/{dest}",
```

Comment:
I have done so although, it looks like `copyTo` has fallen into disgrace in favor of `rewriteTo` but I cannot guarantee it's not used anymore, or that it will not be used again in a minor version update of the lib.